<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AtlasEditorUI</class>
 <widget class="QMainWindow" name="AtlasEditorUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Elang Atlas Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="actionNewAtlas"/>
    </widget>
    <widget class="QMenu" name="menuOpen">
     <property name="title">
      <string>Open</string>
     </property>
     <addaction name="actionOpenTexture"/>
     <addaction name="actionOpenAtlas"/>
    </widget>
    <addaction name="menuNew"/>
    <addaction name="menuOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSaveAtlas"/>
    <addaction name="actionSaveAtlasAs"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionCellsView"/>
    <addaction name="actionPivotView"/>
    <addaction name="actionClipsView"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewAtlas"/>
   <addaction name="separator"/>
   <addaction name="actionOpenTexture"/>
   <addaction name="actionOpenAtlas"/>
   <addaction name="separator"/>
   <addaction name="actionSaveAtlas"/>
   <addaction name="separator"/>
   <addaction name="actionCellsView"/>
   <addaction name="actionPivotView"/>
   <addaction name="actionClipsView"/>
  </widget>
  <action name="actionNewAtlas">
   <property name="text">
    <string>Atlas...</string>
   </property>
   <property name="iconText">
    <string>New Atlas</string>
   </property>
   <property name="toolTip">
    <string>New Atlas</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpenTexture">
   <property name="text">
    <string>Texture...</string>
   </property>
   <property name="iconText">
    <string>Open Texture</string>
   </property>
   <property name="toolTip">
    <string>Open Texture</string>
   </property>
  </action>
  <action name="actionOpenAtlas">
   <property name="text">
    <string>Atlas...</string>
   </property>
   <property name="iconText">
    <string>Open Atlas</string>
   </property>
   <property name="toolTip">
    <string>Open Atlas</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveAtlas">
   <property name="text">
    <string>Save Atlas</string>
   </property>
   <property name="iconText">
    <string>Save Atlas</string>
   </property>
   <property name="toolTip">
    <string>Save Atlas</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCellsView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cells View</string>
   </property>
  </action>
  <action name="actionPivotView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pivot View</string>
   </property>
   <property name="toolTip">
    <string>Pivot View</string>
   </property>
  </action>
  <action name="actionClipsView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Clips View</string>
   </property>
  </action>
  <action name="actionSaveAtlasAs">
   <property name="text">
    <string>Save Atlas As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
